### DISCUSSION TITLE
Show commit message before tags (or hide tags) in the commits panel

### DISCUSSION THREAD

**Discussion by jpaju** [jesseduffield/lazygit#3294] [Q&A]
I'm encountering a challenge in repositories where commits are often tagged with multiple tags. This results in the commit messages being truncated, leaving only a few characters visible in the commits panel. I am looking for a way to either hide these tags in the commits panel or display them after the commit message to ensure the commit messages are readable.

So far, I've explored the branchLogCmd and allBranchesLogCmd settings in the configuration, but I haven't been successful in altering the appearance of the commits in the panel. I am not entirely sure how these configurations affect the display.

TLDR; Is it possible to configure the commit format in the commit panel in a way that commit message would be before tags ?

**Comment by rag-hav** [comment]
Facing the same issue.

**Reply by dima-lurie** [reply]
same issue. this is frustrating, as i really love lazygit

**Reply by DiegoRosales** [reply]
I also have that issue. I can't see the commit message at all. Only tags.

**Comment by Cway14** [comment]
I experience this issue as well. I have opened a PR to add a configuration item that hides git tags: https://github.com/jesseduffield/lazygit/pull/4629

**Reply by Cway14** [reply]
Though a better fix would be able to fully customize the commit logs.
